packages/backend-common/config.d.ts

```diff
@@ -20,6 +20,16 @@ export interface Config {
   };
 
   backend: {
+    /** Backend configuration for when request authentication is enabled */
+    authorization?: {
```

Comment:
should this be `authentication`?

Reply:
Yeah feeling that too. Given that we already have the top-level `auth` block, maybe we could just make it `auth` as I would hate to split the blocks if we end up adding authorization features as well in the future?

Reply:
Not sure why @Rugvip's comment is not showing up in thread here:
> Yeah feeling that too. Given that we already have the top-level auth block, maybe we could just make it auth as I would hate to split the blocks if we end up adding authorization features as well in the future?